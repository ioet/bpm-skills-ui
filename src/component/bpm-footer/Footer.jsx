import React from 'react';
import { withStyles } from '@material-ui/core/styles';
import Button from '@material-ui/core/Button/Button';
import PropTypes from 'prop-types';
import { FooterConst } from './FooterConstants';
import { FooterStyles } from './FooterStyles';

const Footer = (props) => {
  const { classes } = props;

  return (
    <div className={classes.root}>
      <Button color="primary" href={FooterConst.BPM_GITHUB} className={classes.button} target="_blank">
        {FooterConst.LEARN_MORE}
      </Button>
    </div>
  );
};

Footer.propTypes = {
  classes: PropTypes.object.isRequired,
};

export default withStyles(FooterStyles)(Footer);
