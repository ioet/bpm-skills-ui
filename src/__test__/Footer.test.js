import React from 'react';
import Footer from '../component/presentational/Footer';

describe('<Footer />', () => {
  test('Component Renders correctly', () => {
    const appWrapper = shallow(<Footer />);

    expect(toJson(appWrapper))
      .toMatchSnapshot();
  });
});
