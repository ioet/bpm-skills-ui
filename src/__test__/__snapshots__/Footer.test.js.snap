// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> Component Renders correctly 1`] = `
<Footer
  classes={
    Object {
      "button": "Footer-button-2",
      "root": "Footer-root-1",
    }
  }
/>
`;
